src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -17,7 +17,7 @@
 public class DriverDaoImpl implements DriverDao {
     @Override
     public Driver create(Driver driver) {
-        String query = "INSERT INTO drivers (name, license_number) VALUES (?, ?)";
```

Comment:
I don't understand why you have to change name of columns in DB to you variables names?